communications/rfc.rst

```diff
@@ -177,6 +177,12 @@ If an RFC includes API changes, including deprecations, or updates to a data mod
 
 In all of these cases, the RFC is marked as **Flagged** to indicate that it is being considered by the DMCCB.
 
+As a result of the discussion on a **Flagged** RFC, the DMCCB may decide to approve it and move forward with the proposal.
```

Comment:
The "may" here implies that we should say what happens if DMCCB do not approve.

Reply:
Fine, I add some text about what happens if the DMCCB do not approve the change.